docs/scos/dev/glue-api-guides/202307.0/data-exchange-api/how-to-guides/how-to-configure-data-exchange-api.md

```diff
@@ -0,0 +1,132 @@
+      "fieldVisibleName": "id_country",
+      "isEditable": false,
+      "isCreatable": false,
+      "type": "integer",
+      "validation": {
+        "isRequired": false
+      }
+    },
+    {
+      "fieldName": "iso2_code",
+      "fieldVisibleName": "iso2_code",
+      "type": "string",
+      "isEditable": true,
+      "isCreatable": true,
+      "validation": {
+        "isRequired": true,
+        "maxLength": 2,
+        "minLength": 2
+      }
+    }
+  ]
+}
+```
+
+The following table describes the purpose of each field:
+
+| FIELD | SPECIFICATION |
+| --- | --- |
+| identifier | The name of the column in the table that serves as an identifier. It can be any chosen column from the table, typically used as a unique ID for each record. |
+| fields | An array containing the descriptions of the columns from the table. It defines the columns that are included for interaction. The API exposes and operates on the included columns. Columns that are not included in the array are inaccessible through API requests. |
+| fieldName | The name of the column in the database table. |
+| fieldVisibleName | The name used for interacting with the field through API requests. It's a more user-friendly and descriptive name compared to the physical column name. |
+| type | The data type of the field. It specifies the expected data format for the field, enabling proper validation and handling of values during API interactions. |
+| isEditable | Defines if the field's value can be changed. |
+| isCreatable | Defines if the field can be included in requests to provide an initial value during record creation. |
+| validation | Contains validation configurations. Proper validation ensures that the provided data meets the specified criteria. |
+| required | Defines if the field must be provided in API requests. |
+| maxLength/minLength | Defines the minimum/maximum length allowed for the field with a string type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. |
+| max/min | Defines the minimum/maximum value allowed for the field with an integer type. It enforces a lower boundary, ensuring that the field's value meets or exceeds the defined minimum requirement. Optional. |
+
+
+{% info_block infoBox %}
+
+* We recommend setting `isEditable` and `isCreatable` to `false` for fields that serve as identifiers or keys, ensuring their immutability and preserving the integrity of the data model.
+
+* For the fields with numerable values, an integer data type is a non-decimal number between -2147483648 and 2147483647 in 32-bit systems and, in 64-bit systems, between -9223372036854775808 and 9223372036854775807. If you need or anticipate values outside of this range, you can set the value as a string type.
+
+* The Data Exchange API supports the following types for the configured fields:
+  * boolean
+  * integer
+  * string
+  * decimal
+
+{% endinfo_block %}
+
+## Create Data Exchange API endpoints
```

Comment:
How is the info in this section related to the info in the previous section?

Reply:
I would rename it to `Create Data Exchange API Configuration`.
The part above is about configuration fields and what they stand for. 
The second part explains how to enable `/dynamic-data/country` endpoint and configure it based on those fields.

Reply:
Before taking the steps in this section, should I configure the endpoint by following [Configuration of Data Exchange API endpoints](https://github.com/spryker/spryker-docs/pull/2108/files#diff-3955fbd09ad2d490f8edb5870ee52e824b3e0fc26861af7fc3293f946dda27eeR12)?

Reply:
Sure, it won't work without that.